src/ui/client/src/components/Content.js

```diff
@@ -1,12 +1,13 @@
 import React from "react";
-import {Container, Row, Col, Dropdown, DropdownToggle, DropdownMenu, DropdownItem, Navbar, NavbarBrand} from 'reactstrap';
-import ReactMapboxGl, { Layer, Feature } from 'react-mapbox-gl';
+import {Container, Row, Col, Navbar, NavbarBrand} from 'reactstrap';
+import ReactMapboxGl from 'react-mapbox-gl';
+import Autosuggest from 'react-autosuggest';
 
 const API_BASE_URL = "http://localhost:4000"; //process.env.API_URL;
 
 const Map = ReactMapboxGl({
     accessToken:
-      'pk.eyJ1IjoibWFnaWNtYXgzMiIsImEiOiJjanR0MGxubXcweHVzNDVwaWV2MGQ1ZWp3In0.8YiTJ8uuaq831eLcsjc03w'
+      'pk.eyJ1IjoibWFnaWNtYXgzMiIsImEiOiJjanR0MGxubXcweHVzNDVwaWV2MGQ1ZWp3In0.8YiTJ8uuaq831eLcsjc03w' //process.env.Mapbox_Access_Token;
```

Comment:
Should we think of a way to remove the access tokens from our source?

Reply:
Yes. I shouldn't have had it in there. We can incorporate it as an environment variable (which is why I added that comment)

Reply:
Already merged. just to make sure it stays fresh in our mind.